doc/tutorials/sagemaker/sme_train_model.ipynb

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
<p>If we want to test the notebook later automatically, we need to implement the poll loop, instead of entrust the polling the user</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I suggest we make another ticket for this.

Reply:
<p>yes, please create a ticket for it.</p>